Library/Homebrew/dev-cmd/extract.rb

```diff
@@ -0,0 +1,126 @@
+#:  * `extract` [`--force`] <formula> <tap> [`--version=`<version>]:
+#:    Looks through repository history to find the <version> of <formula> and
+#:    creates a copy in <tap>/Formula/<formula>@<version>.rb. If the tap is
+#:    not installed yet, attempts to install/clone the tap before continuing.
+#:
+#:    If `--force` is passed, the file at the destination will be overwritten
+#:    if it already exists. Otherwise, existing files will be preserved.
+#:
+#:    If an argument is passed through `--version`, <version> of <formula>
+#:    will be extracted and placed in the destination tap. Otherwise, the most
+#:    recent version that can be found will be used.
+
+require "utils/git"
+require "formulary"
+require "tap"
+
+class BottleSpecification
```

Comment:
@alebcay Use `# rubocop:disable Style/MethodMissingSuper` above this line to fix `brew style`. Use `# rubocop:enable Style/MethodMissingSuper` above `module Homebrew` to re-enable it there. Thanks!